lib/broccoli/ember-app.js

```diff
@@ -956,18 +956,26 @@ EmberApp.prototype.appAndDependencies = function() {
   });
 };
 
+EmberApp.prototype.test = function() {
```

Comment:
* I can add docs
* The name is meant to be consistent with the names of the other methods feeding into toArray() (like javascript()) but I can rename to something more descriptive.